app/gui/src/project-view/components/ComponentBrowser/ComponentList.vue

```diff
@@ -42,18 +47,34 @@ const displayedSelectedComponentIndex = computed({
   },
 })
 
-watch(toRef(props, 'filtering'), () => (displayedSelectedComponentIndex.value = 0))
+const filtering = computed(() => {
+  const currentModule = projectStore.moduleProjectPath
+  return new Filtering(props.filter, currentModule?.ok ? currentModule.value : undefined)
```

Comment:
This might be a little simpler with `unwrapOr`.

Reply:
currentModule may be `undefined`, so I wouldn't avoid `?:`